docker-compose.yaml

```diff
@@ -1,36 +1,19 @@
-version: '3'
+version: '3.4'
 
 services:
-    composer:
+    app:
         build:
             context: .
-            dockerfile: Dockerfile.composer
-            args:
-                composer_dev_arg: ${COMPOSER_DEV_ARG:- }
-        image: libero/browser_composer:${IMAGE_TAG:-master}
-        command: /bin/true
-    fpm:
-        build:
-            context: .
-            args:
-                image_tag: ${IMAGE_TAG:-master}
+            target: prod
+        environment:
+            APP_SECRET:
         image: libero/browser:${IMAGE_TAG:-master}
-        depends_on:
-            - composer
     web:
-        image: nginx:1.15.2-alpine
+        image: nginx:1.15.5-alpine
         volumes:
-            - .docker/nginx.conf:/etc/nginx/conf.d/default.conf
+            - ./.docker/nginx.conf:/etc/nginx/conf.d/default.conf
+            - ./public/:/app/public/
```

Comment:
if there was a sample file in here it would help validate the magical formulas to expose it through nginx

Reply:
Libero favicon?

Reply:
Anything e.g. <img src="https://avatars0.githubusercontent.com/u/37089447?s=200&v=4" />